src/main/java/analysis/emissions/RunEmissionAnalysisByVehicleCategory.java

```diff
@@ -0,0 +1,328 @@
+*Including only required arguments*
+
+/Users/rgraebe/IdeaProjects/matsim-leipzig/output/it-1pct/
+--runId
+leipzig-25pct
+--hbefa-cold
+/Users/rgraebe/shared-svn/projects/matsim-germany/hbefa/hbefa-files/v4.1/EFA_ColdStart_Vehcat_2020_Average_withHGVetc.csv
+--hbefa-warm
+/Users/rgraebe/shared-svn/projects/matsim-germany/hbefa/hbefa-files/v4.1/EFA_HOT_Vehcat_2020_Average.csv
+
+ */
+
+@CommandLine.Command(
+        name = "air-pollution-by-vehicle",
+        description = "Run offline air pollution analysis assuming default vehicles",
+        mixinStandardHelpOptions = true,
+        showDefaultValues = true
+)
+public class RunEmissionAnalysisByVehicleCategory implements MATSimAppCommand {
+
+    private static final Logger log = LogManager.getLogger(RunEmissionAnalysisByVehicleCategory.class);
+
+    @CommandLine.Parameters(paramLabel = "INPUT", arity = "1", description = "Path to run directory")
+    private Path runDirectory;
+
+    @CommandLine.Option(names = "--runId", description = "Pattern to match runId.", defaultValue = "*")
+    private String runId;
+
+    @CommandLine.Option(names = "--hbefa-warm", required = true)
+    private Path hbefaWarmFile;
+
+    @CommandLine.Option(names = "--hbefa-cold", required = true)
+    private Path hbefaColdFile;
+
+    @CommandLine.Option(names = "--hbefa-lookup", description = "Emission detailedVsAverageLookupBehavior", defaultValue = "directlyTryAverageTable")
+    private DetailedVsAverageLookupBehavior lookupBehavior;
+
+    @CommandLine.Option(names = "-p", description = "Password for encrypted hbefa files", interactive = true, required = false)
+    private char[] password;
+
+    @CommandLine.Option(names = "--output", description = "Output events file", required = false)
+    private Path output;
+
+    @CommandLine.Option(names = "--vehicle-type", description = "Map vehicle type to Hbefa category", defaultValue = "defaultVehicleType=PASSENGER_CAR")
+    private Map<String, HbefaVehicleCategory> vehicleCategories;
+
+    @CommandLine.Option(names = "--use-default-road-types", description = "Add default hbefa_road_type link attributes to the network", defaultValue = "true")
+    private boolean useDefaultRoadTypes;
+
+    @CommandLine.Mixin
+    private final CrsOptions crs = new CrsOptions();
+
+    public RunEmissionAnalysisByVehicleCategory() {
+        // You need this constructor, otherwise the psvm-method won't work with <new RunEmissionAnalysisByVehicleCategory().execute(args);>
+    }
+
+    public RunEmissionAnalysisByVehicleCategory(Path runDirectory, String runId, Path hbefaFileWarm, Path hbefaFileCold, Path output) {
+        this.runDirectory = runDirectory;
+        // e.g. "/Users/rgraebe/IdeaProjects/matsim-leipzig/output/it-1pct"
+        this.runId = runId;
+        // e.g. "leipzig-25pct"
+        this.hbefaWarmFile = hbefaFileWarm;
+        // e.g. "/Users/rgraebe/shared-svn/projects/matsim-germany/hbefa/hbefa-files/v4.1/EFA_HOT_Vehcat_2020_Average.csv"
```

Comment:
same here: no one needs our local paths/files :) .... and  please talk to @rakow how to use the encrypted files from the public-svn ... 
I am on vacation next week, so I cannot show it to you before that. (@rakow : If you do not find the time until I am back, I can also show it to @rgraebe after when I am back)